xbmc/network/websocket/WebSocketV13.cpp

```diff
@@ -102,7 +104,11 @@ bool CWebSocketV13::Handshake(const char* data, size_t length, std::string &resp
 
   // There must be a "Connection" header with the value "Upgrade"
   value = header.getValue(WS_HEADER_CONNECTION_LC);
-  if (value == NULL || strnicmp(value, WS_HEADER_UPGRADE, strlen(WS_HEADER_UPGRADE)) != 0)
+  if (value == nullptr)
+    elements = std::vector<std::string>();
```

Comment:
Why is this necessary? `elements` has been declared and initialized as `std::vector<std::string>` already.

Reply:
just wanted to make it safe for the future, but I'll move elements declaration down here